app/views/play/menu/InventoryModal.coffee

```diff
@@ -146,8 +149,11 @@ module.exports = class InventoryModal extends ModalView
       @itemGroups.restrictedItems.add(item)
       item.classes.push 'restricted'
     else if subscriber
-      @itemGroups.subscriberItems.add(item)
-      item.classes.push 'subscriber'
+      if me.isStudent()
+        @itemGroups.availableItems.add(item)
+      else
+        @itemGroups.subscriberItems.add(item)
+        item.classes.push 'subscriber'
```

Comment:
Should instead just make it so you are not a subscriber if you're a student, where `subscriber` is defined.